specificatie/openapi.yaml

```diff
@@ -604,18 +604,23 @@ components:
           description: |
                         Leeftijd in jaren op het moment van bevragen.
           example: 34
-        naam:
-          $ref: "#/components/schemas/Naam"
         inOnderzoek:
           $ref: "#/components/schemas/PersoonInOnderzoek"
-        geboorte:
-          $ref: "#/components/schemas/Geboortedatum"
-        opschortingBijhouding:
-          $ref: "#/components/schemas/OpschortingBijhoudingBasis"
-        overlijden:
-          $ref: "#/components/schemas/OverlijdenBasis"
-        verblijfplaats:
-          $ref: "#/components/schemas/VerblijfplaatsBeperkt"
+    IngeschrevenPersoonBeperkt:
+      allOf:
+        - $ref: "#/components/schemas/IngeschrevenPersoonBasis"
+        - type: "object"
+          properties:
+            naam:
+              $ref: "#/components/schemas/Naam"
+            geboorte:
+              $ref: "#/components/schemas/Geboortedatum"
+            opschortingBijhouding:
+              $ref: "#/components/schemas/OpschortingBijhoudingBasis"
+            overlijden:
+              $ref: "#/components/schemas/OverlijdenBasis"
```

Comment:
Is het bij de ingeschrevenPersoonBeperkt niet relevant of Overlijden of OpschortingBijhouding in onderzoek is ? 
Dat is niet consistent met hoe er met Naam, Geboortedatum en Verblijfplaats is omgegaan.

Reply:
voor overlijden nemen we alleen indicatieOverleden op. we hebben ooit besloten dat inOnderzoek getoond wordt op velden die je vraagt/krijgt en niet op daarvan afgeleide velden. Dus als datum overlijden in onderzoek staat, maar je vraagt alleen indicatieOverleden op, krijg je geen inOnderzoek. Dat is zo bij het raadplegen van de persoon. Bij het zoeken kan je niet meer krijgen en is inOnderzoek dus niet nodig.

Reply:
opschortingBijhouding heeft geen inOnderzoek, want dit kan niet in onderzoek staan. Ook nu in v1.3 bij het raadplegen van een persoon kan je bij opschortingBijhouding geen inOnderzoek krijgen